contracts/src/v0.8/llo-feeds/v0.4.0/DestinationFeeManager.sol

```diff
@@ -551,5 +551,6 @@ contract DestinationFeeManager is IDestinationFeeManager, ConfirmedOwner, TypeAn
   function setRewardManager(address rewardManagerAddress) external onlyOwner {
     if (rewardManagerAddress == address(0)) revert InvalidAddress();
     s_rewardManager = IDestinationRewardManager(rewardManagerAddress);
+    IERC20(i_linkAddress).approve(address(s_rewardManager), type(uint256).max);
```

Comment:
@Fletch153  we do this when we set a reward manager in the constuctor. I think we also need to do it here in this function.

we probably need a unit test for this. any suggestions ?

Reply:
good spot!